aiidalab_widgets_base/computational_resources.py

```diff
@@ -892,24 +900,18 @@ def on_setup_computer(self, _=None):
         )
         try:
             computer = computer_builder.new()
+            computer.store()
         except (
             ComputerBuilder.ComputerValidationError,
             common.exceptions.ValidationError,
         ) as err:
             self.message = f"{type(err).__name__}: {err}"
             return False
 
-        try:
-            computer.store()
-        except common.exceptions.ValidationError as err:
-            self.message = f"Unable to store the computer: {err}."
-            return False
-
-        if self._configure_computer(computer):
```

Comment:
It would be good if you could verify that your new tests fail when this line is deleted. :-)

Reply:
Great! that exactly manifest the bug. Add the configure function back to solve the issue. You can find the change from last commit https://github.com/aiidalab/aiidalab-widgets-base/pull/401/commits/1c568d38db453712c4f2c11ea862bb0621b036de

I also change a bit of the setup-configure flow, I combine the computer "setup" and "configure" in the block of a try, which only store the computer node when the setup and the configure are both successful. I did this because we only provide the widget called "Computer Setup" but no "computer configure". When the computer is setup and stored but the configure phase is not finished. The computer in the database has the UUID but leaves it in a very strange state and can not be used for further code setup.